src/shared/lib/timers/hooks/useAppTimer.ts

```diff
@@ -15,7 +17,10 @@ const useAppTimer = (appTimerConfig: AppTimerConfig) => {
   callbacksRef.current = { onFinish };
 
   useEffect(() => {
-    const onTimerEnd = () => callbacksRef.current.onFinish();
+    const onTimerEnd = (): TimerCallbackFeedback => {
+      callbacksRef.current.onFinish();
+      return 'no-feedback';
+    };
```

Comment:
Why isn't type `void` enough?

Reply:
To make the code more robust and clear. In case of `void` is very simple to forget to return needed result.

Reply:
It is not very clear why a callback should return a 'no-feedback' string though.

Reply:
reverted